tests/time_tests/test_runner/test_timetest.py

```diff
@@ -43,7 +43,7 @@ def test_timetest(instance, executable, niter, cl_cache_dir, model_cache, model_
     :param niter: number of times to run executable
     :param cl_cache_dir: directory to store OpenCL cache
     :param cpu_cache: flag to enable model CPU cache
-    :param vpu_compiler: flag to change VPUX compiler type
+    :param vpu_compiler: flag to change NPU compiler type
```

Comment:
Please update device name from VPUX to NPU.

Reply:
Done.